aks/application/variables.tf

```
@@ -45,6 +51,12 @@ variable "kubernetes_secret_name" {
   description = "Name of the Kubernetes secrets"
 }
 
+variable "kubernetes_id" {
```

```suggestion
variable "kubernetes_cluster_id" {
```

I wonder if this is a better name as many objects in Kubernetes can have IDs, although we've got a description at first glance it's not immediately obvious within the application module which ID this would be referring to.

I think it's fine to leave the output as `kubernetes_id` from within the cluster data module as we know that module is responsible for the cluster.

---

changed variable to kubernetes_cluster_id